lib-multisrc/madara/build.gradle.kts

```diff
@@ -2,7 +2,7 @@ plugins {
     id("lib-multisrc")
 }
 
-baseVersionCode = 35
+baseVersionCode = 36
```

Comment:
not too important so don't bump version

Reply:
on another PR earlier, the change was even less significant but @beerpiss said need to bump version no matter how small change is. It's kind of hard to understand :)

Reply:
What beer said holds true but Madara is too big to push update every day